configs/defaults/gcnv.toml

```diff
@@ -0,0 +1,7 @@
+[workflow]
+intervals_path = 'gs://cpg-common-main/references/hg38/v0/exome_calling_regions.v1.interval_list'
+# TODO Where can this file come from, and does it need to vary
+ploidy_priors = 'gs://cpg-common-main/ploidy-priors-TODO.tsv'
```

Comment:
Similarly there's a ploidy priors under

```toml
[references.gatk_sv]
...
contig_ploidy_priors = "gs://cpg-common-main/references/gatk-sv/hg38/v0/sv-resources/resources/v1/hg38.contig_ploidy_priors_homo_sapiens.tsv"
```

I'm not sure if this differs from the one required for exomes

Reply:
Yes, I tracked that one down by looking in the bucket. It's a simple enough file that I think it'll be fine for exomes.